packages/documentation/src/stories/internet-header/getting-started.stories.mdx

```diff
@@ -61,9 +61,13 @@ If you are not using any loader or don't want to install from npm, you can load
 
 ## Configuration
 
-The Internet Header is built to support the already existing Portal Configuration used for the old Internet Header. If you already have a configuration, you don't have to change it. If you are working on a new project, you can order a new configuration with the [Post Portal Team](mailto:digitalconsulting@post.ch).
+The Internet Header is built to support the already existing Portal Configuration used for the old Internet Header.
 
-Not every Online Service has configurations for all environments. For example, "TOPOS" only has `int02` configured and cannot use `int01` as value for environment. If you are not sure what's configured for your project, you can contact [digitalconsulting@post.ch](mailto:digitalconsulting@post.ch).
+If you're working on a new project, you probably need a new configuration. Please click the button below.
+
+<a class="mb-small-r btn-primary btn btn-rg btn-animated" href="mailto:digitalconsulting@post.ch" target="_blank" rel="noopener"><span>Contact Post Portal Team</span></a>
+
+Not every Online Service has configurations for all environments. For example, "TOPOS" only has `int02` configured and cannot use `int01` as value for environment. If you are not sure what's configured for your project, please also click the button above.
```

Comment:
```suggestion
Not every Online Service has configurations for all environments. For example, "TOPOS" only has `int02` configured and cannot use `int01` as value for environment. If you are not sure what's configured for your project, please also use the button above, to ask the post-portal team about it.
```

Reply:
Thanks for the suggestion, although the comma after "above" isn't really fitting. What do you think of leaving it out?